### ISSUE TITLE
Log file custom path

### ISSUE THREAD

**Issue by jaredballou** [djfdyuruiry/cnc-td-nco-mod#37]
I need to put the log file outside of the My Documents path.

Do we want to simply have a check on the LogFile rule to use it as an absolute path when prefixed with a drive letter, or have a separate rule named LogFilePath that allows an override?

**Comment by djfdyuruiry** [comment]
Yeah I think just detect if it's an absolute path and use that as the entire path if so
